Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -14,6 +14,8 @@
 limitations under the License.
 */
 
+using System;
```

Comment:
Remove new (and presumably extraneous) using directive?

Reply:
Oops, yes, done.